pkg/support/dump.go

```diff
@@ -37,6 +37,9 @@ import (
 	"antrea.io/antrea/pkg/util/logdir"
 )
 
+// Declared as variables for testing.
+var exe = exec.New()
```

Comment:
This is used to create new `exe = new(testExecuter)` (for testing purpose in dump_test.go and dump_windows.go)

Reply:
this is one of the unit tests:
```go
	exe = new(testExec)
	fs := afero.NewOsFs()
	defer func() {
		exe = exec.New()
	}()
	for _, tc := range tests {
		t.Run(tc.name, func(t *testing.T) {
			varr := NewAgentDumper(fs, exe, nil, nil, nil, tc.since, tc.v4Enabled, tc.v6Enabled)
			err := varr.DumpHostNetworkInfo(tc.basedir)
			if tc.expectedErr != "" {
				require.EqualError(t, err, tc.expectedErr)
			} else {
				assert.NoError(t, err)
			}
		})
	}
```

I don't see why you need a global variable here. The executor is passed as an argument to `NewAgentDumper`. The above test could be simplified as follows:

```go
	exe := new(testExec)
	fs := afero.NewOsFs()
	for _, tc := range tests {
		t.Run(tc.name, func(t *testing.T) {
			varr := NewAgentDumper(fs, exe, nil, nil, nil, tc.since, tc.v4Enabled, tc.v6Enabled)
			err := varr.DumpHostNetworkInfo(tc.basedir)
			if tc.expectedErr != "" {
				require.EqualError(t, err, tc.expectedErr)
			} else {
				assert.NoError(t, err)
			}
		})
	}
```